spec/rubocop/cop/rake/desc_spec.rb

```diff
@@ -63,4 +63,10 @@
       task.name
     RUBY
   end
+
+  it 'does not register an offense for prerequisite declarations' do
+    expect_no_offenses(<<~RUBY)
+      task release: 'changelog:check_clean'
```

Comment:
I'm not sure if this change makes sense. It can be a front end task for multiple (dependent) tasks.

```ruby
desc "Build MySQL and PostgreSQL test databases"
task create: ["db:mysql:build", "db:postgresql:build"]
```

If this cop always allows the use case, it cannot be detected.
How about detecting when there is only one dependent task?

Reply:
@koic I have changed to not register offense for multiple dependents. Please have a look.